manifests/kustomize/env/cert-manager/dev/kustomization.yaml

```diff
@@ -6,6 +6,7 @@ namespace: kubeflow
 resources:
 - ../../dev
 - ../base
+- ../base/webhook
```

Comment:
Instead of this, could you have `manifests/kustomize/env/cert-manager/base/kustomization.yaml` have a `resources` entry for `manifests/kustomize/env/cert-manager/base/webhook`? Then anything already using `manifests/kustomize/env/cert-manager/base` will also get the webhook manifests.